<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <title>Bootstrap</title>
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link href="https://getbootstrap.com/docs/4.4/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
    <link href="style.css" rel="stylesheet">
</head>

<body>
    <header>
        <h1>Pantograph Introduction</h1>
        <h2>COVID-19 Variants Analysis</h2>
    </header>
    <main>

        <div class="container-fluid">
            <div class="row">
                <div class="col-sm-12">
                    <div class="content">

                        <h1 id="pantograph-project">Pantograph Project</h1>
                        <p>A Scalable Approach to Pangenome Visualization</p>
                        <h2 id="covid-19-foreword">COVID-19 Foreword</h2>

                        <p>Pantograph as a project started in 2018 as a way to unlock the next level of population genetics for researchers. Graph Genomes discussed here are a new way of capturing sequence data designed to fix problems systemic to the technology that we've been using for the past 30 years.</p>

                        <p>For example, reference bias means genetic analysis is more accurate for Europeans than for Africans (Liverpool 2019). A technical paper on Pantograph was being written by a team of 10 scientists before COVID-19 hit.</p>

                        <p>Pantograph is extremely relevant to the current pandemic because the success or failure of our efforts to fight the disease rely upon the sequence diversity of the virus itself. Tests for infection rely on knowing the exact sequence being tested. A rearrangement in the order of genes, even if the content is the same, will return a false negative test.</p>

                        <p>Second, the vaccine targeting the Spike protein on the outside of the virus relies on a lack of genetic diversity in the Spike protein sequence (NIH clinical trial of investigational...). If there are any strains with a mutant protein, the vaccine could be rendered ineffective and the virus would continue to spread.</p>

                        <p>SARS-CoV-2 is an RNA virus that will likely infect billions of people, giving it a much higher mutation capability than we've previously dealt with in pandemics. For example, the common cold is so impossible to eradicate precisely because of the number of people infected and thus the high number of mutations which exist around the globe.</p>

                        <p>Current sequencing techniques may be under-representing the full sequence diversity of the virus because they are reference based. Eliminating reference bias and enabling species genetic diversity on thousands of individuals is the core goal of using a graph genome.</p>

                        <p>Pantograph is a very small piece in a worldwide effort to eradicate this disease. It is by no means the most important and the disease will likely end without any involvement on our part.</p>

                        <p>However, given the scale of the pandemic, even a tiny improvement or speedup can result in thousands of lives saved. That's a difference which is worth investing our resources in. Pantograph will continue to be useful in a wide range of disease application after the current crisis is averted, so that we are never caught unprepared again.</p>

                        <h2 id="project-overview">Project Overview</h2>
                        <p>Linear reference genome representations in current use cannot capture non- reference sequence variation when visualizing hundreds of individuals. Genome graphs are an attractive solution to visualize and investigate the whole variation within large sets of individuals at glance.</p>

                        <p>Structural variants are associated with many diseases, including childhood cancers and schizophrenia. In order to make sequencing data actionable for clinicians, structural variants need to be put in the larger context of all known genetic variation.</p>

                        <p>Understanding genetic variation in plants is equally important for identifying pest and pathogen resistance and increasing crop yields to support a growing population.</p>

                        <h2 id="visualization">Visualization</h2>

                        <p>Here we present Pangenome Schematics, the first visualization with the scalability to render graph genomes from thousands of individuals over gigabase genome sizes with the ability to show both whole chromosome features as well as zoom into nucleotide sequence variation.</p>

                        <p>No tool to date has satisfied all these criteria. Scalability is accomplished through graph sorting and binning adjacent sequences to create shared syntenic blocks called Components. Non- linear structural variants can be shared by many individuals as Links across the pangenome.</p>

                    </div>
                </div>
            </div>
        </div>

    </main>

    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
</body></html>

